RELEASE.md

```diff
@@ -36,31 +36,30 @@ All future changes adopted into master will need to be back-ported to those bran
 
 ## Process
 
-### Testing Release (Weekly, Thursday)
+### Testing Release (Second and Forth Thursday)
 
-Each _**Thursday**_, this repository should be tagged for SOAK testing as follows:
+On the second and forth _**Thursday**_, this repository should be branched for SOAK testing by setting the `testing` branch to the head of master and force-pushing.
 
-- Using automation, parse the PR logs for release notes and generate and commit a Changelog.md
-- One _**tag**_ is made for the each supported version of Kubernetes with an incremented release counter
-- If the current version of Kubernetes is `1.17.2`, and the last release was `stable-1.17-5`, the new SOAK tag will be `testing-1.17-6`.
-- For the previous Kubernetes version, the last release may have been `stable-1.16-9`. The new tag for this Kubernetes version is `testing-1.16-10`.
-- The oldest supported release similarly might be `testing-1.15-27` for a prior `stable-1.15-26`.
-- These tag versions, in the form of `major.minor-dist`, only the `major.minor` refer to the kubernetes version. The api within a minor version should not be changing so there should never be a need to refer to the kubernetes patch version.
+- This set of Addons are installed into the SOAK cluster.
 
-**NOTE:** If a breaking change causes a diversion from an older release of Kubernetes to a newer one, prior to tagging the older version must be branched, ie. `stable-1.16-9` would become `stable-1.16`, the changes since the `stable-1.16-9` tag would be merged into this branch, and the new _tag_ would still be `testing-1.16-10` but pointing to the last change on the `stable-1.16` branch.
+### Stable Release (Second and Forth Wednesday)
 
-- This set of Addons are installed into a SOAK cluster<sup>[3](#footnote3)</sup>.
-
-### Stable Release (Weekly, Wednesday)
-
-Each _**Wednesday**_:
+On the second and forth _**Wednesday**_:
 
+- Using automation, parse the PR logs in the `testing` branch for release notes and generate and commit a Changelog.md
+- One _**tag**_ is made for the each supported version of Kubernetes with a consistent semver suffix
+- If the current version of Kubernetes is `1.17.2`, and the last release was `stable-1.17-1.2.x`, the new SOAK tag will be `release-1.17-1.3.0`
```

Comment:
this might not always the case. e.g. when breaking changes occur, this example would be `release-1.17-2.0.0`